configs/acra-censor.example.yaml

```diff
@@ -1,4 +1,5 @@
 ignore_parse_error: false
+parse_errors_log: unparsed_queries_log
```

Comment:
what about `uparsed_queries.log` name? to add some common extension to avoid meaning some binary file without extension on linux